### ISSUE TITLE
Image tracking fails after device orientation has changed

### ISSUE THREAD

**Issue by DouglasLapsley** [hiukim/mind-ar-js#212]
Please see [here](https://hiukim.github.io/mind-ar-js-doc/examples/basic) on iPhone.

- Load in either landscape or portrait orientation
- Image tracking works
- Rotate the device to change orientation
- Image tracking now fails with object rotated and distorted.
- Rotate back to original orientation and image tracking works again

Is there a re-init method that can be called on orientationchange or something to deal with this?

Thank you.

**Comment by arslc** [comment]
@DouglasLapsley  try this:

```javascript
window.addEventListener('orientationchange', () => {
 //stop mindar
 mindar.stop()

 // ur function to hide 3D object (optional)

 //restart mindar after 1 second delay
 setTimeout(()=>{
  mindar.start()
 }, 1000)

}, false)
```
Not the most elegant way, but as a temporary solution it may help.

*P.S. I know, this is deprecated event, but it still work on iOS 16.
So we can use that until @hiukim not fix this bug*

**Comment by hiukim** [comment]
fixed in v1.2.2
